tests/test_currency_network_fees.py

```diff
@@ -5,7 +5,7 @@
 import eth_tester.exceptions
 from tldeploy.core import deploy_network
 
-from .conftest import EXTRA_DATA
+from .conftest import EXTRA_DATA, EXPIRATION_TIME
```

Comment:
my recommendation would be to never import from conftest.py. (I know the import was there before, so you can keep it like it is). but this will make things harder if you want a second conftest.py file.

Reply:
The only solution is to create a fixture and use it in every test, or is there better? (i could not find better by googling)

Reply:
You could also create a testing package from which you import.